doc/source/user/basics.indexing.rst

```diff
@@ -29,9 +29,9 @@ Note that in Python, ``x[(exp1, exp2, ..., expN)]`` is equivalent to
 for the former.
 
 
-.. _basic-slicing-and-indexing:
+.. _basic-indexing:
```

Comment:
This link may be in use by others outside NumPy. Could we leave both forms with a comment "legacy link name"?

Reply:
I added this link in this PR itself in a previous commit because I needed it for the stub Indexing doc that was left behind in the reference guide. Since then I removed the stub completely but forgot to remove this link... So should I just remove this link altogether?

Reply:
Ahh, I see. If it is new and no-one needs it then it can be removed.